mir-module/src/main/resources/xsl/metadata/static/mir-admindata-box-static.xsl

```diff
@@ -30,10 +30,12 @@
             <xsl:with-param select="mycoreobject/service/servdates/servdate[@type='createdate']" name="nodes"/>
             <xsl:with-param select="i18n:translate('metaData.createdAt')" name="label"/>
           </xsl:call-template>
-          <xsl:call-template name="printMetaDate">
-            <xsl:with-param select="mycoreobject/service/servflags/servflag[@type='createdby']" name="nodes"/>
-            <xsl:with-param select="i18n:translate('metaData.createdby')" name="label"/>
+
```

Comment:
Ideally, both representations (username and real name) are displayed, rather than just one or the other.